.github/workflows/release.yml

```
@@ -0,0 +1,41 @@
+name: Upload Python Package
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    name: Build package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+      - run: pip install --upgrade build
+      - name: Build package
+        run: pyproject-build
```

We need to build a source distribution:

```suggestion
        run: pyproject-build --sdist
```

---

The wheel build doesn't work without additional work (#242).
This situation is unchanged from the current publishing workflow though.